### ISSUE TITLE
Delete a license via REST API

### ISSUE THREAD

**Issue by dineshr93** [fossology/fossology#2114]
### Description

Ability to Delete a license via a REST API

#### Proposal and expected result

input: **license_short_name** at the end of url
```
url = "http://fossology.com:port/repo/api/v1/license/mit"
payload = ""
headers = {
    "accept": "application/json",
    "groupName": "fossy",
    "Authorization": bearer_token
}

response = requests.request("DELETE", url, data=payload, headers=headers)
print(response.text)
```
Output: delete the license

**Comment by SanjaySinghRajpoot** [comment]
I would like to work on this issue. Could you please which files are to be modified?
 Thank You

**Comment by Aditi-Roy-coder** [comment]
I want to work on this issue.

**Comment by thegeekymuggle** [comment]
Hi, I would love to work on the issue.

**Comment by Rahulrajak1710** [comment]
Hi,
I am rahul,a cse undergraduate.I am new to open source contributions but I am well aware of Java,C,Html,Css,JavaScript,React.js.I would love contribute in fossology 
 organization  could you please tell me how to get started?
Hoping to hear from you soon.
Regards
Rahul

**Comment by shaheemazmalmmd** [comment]
@dineshr93 i think it is not safe to delete a license Because, 

- for actual licenses

lets say we have license "X". and some users have already added license "X" in to clearing results
if we delete license "X" then we will end up with errors in that file where license "X" got added. 

So it is not safe delete a license.

- for candidate licenses

we already have a delete functionality which can be reused in REST API.

This functionality will not allow you to directly delete the licenses but it shows you the list of files where this license "X" has been used. so one needs to add a alternate license and remove license "X" then only you would be able to delete it.

**Comment by dineshr93** [comment]
@shaheemazmalmmd yes I agree, there was a use case where EPL-1.0 can be upgraded to EPL-2.0 (due to inbuilt clause) and it's very hard to do it manually.  So I felt the need of this functionality.
